internal/fixture/state.go

```diff
@@ -184,6 +181,10 @@ func (s *State) CompareLive(obj typed.YAMLObject, version fieldpath.APIVersion)
 	return live.Compare(tv)
 }
 
+func (s *State) parseConfig(apiVersion fieldpath.APIVersion, obj typed.YAMLObject) (*typed.TypedValue, error) {
```

Comment:
While I don't disagree that this is useful, this is making that change more complicated than it has to be.

Reply:
Yeah, it's from the rebase. I'll cleanup.